catalyst-gateway/bin/src/settings/mod.rs

```diff
@@ -162,7 +162,7 @@ static ENV_VARS: LazyLock<EnvVars> = LazyLock::new(|| {
 
     let address = StringEnvVar::new("ADDRESS", ADDRESS_DEFAULT.to_string().into());
     let address = SocketAddr::from_str(address.as_str())
-        .inspect(|err| {
+        .inspect_err(|err| {
             error!(
                 "Invalid binding address {}, err: {err}. Using default binding address value {ADDRESS_DEFAULT}.",
```

Comment:
Can we really try and stop using formatted strings for errors?  I get these preexist, but given this PR is directly touching these errors, we could be making them structured errors at the same time.  Humans are not reading these messages, our log ingesting process is.  And its much easier to search/automate log messages based on structured fields than chatty humanized messages.

Something like: `socket_err: err, invalid_addr: addr, default: ADDRESS_DEFAULT, "Socket Error binding address.  Using Default"`